### PR TITLE
Test requirements

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

**Comment by tisnik** [comment]
[test]

**Comment by abs51295** [comment]
By running `pip-compile` again on new `requirements.in`, this is the updated `requirements.txt`: https://gist.github.com/abs51295/f2caddf46e9435b8311bd7c5b51125f2. Can you update that as well?

**Comment by tisnik** [comment]
@abs51295 WDYT?

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
@abs51295 there's still some problem with building one package. let me check what's wrong. Seems like`make` is missing, but it's installed.

**Comment by abs51295** [comment]
@tisnik you should install make here: https://github.com/fabric8-analytics/fabric8-analytics-stack-analysis-base/blob/master/Dockerfile

**Comment by tisnik** [comment]
@abs51295 yeah, I did it ;) btw why do you need to install tkinter et al? IIRC g++ and python-devel would be enough.

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,10 @@ LABEL maintainer="Avishkar Gupta <avgupta@redhat.com>"
 # install python packages
 # --------------------------------------------------------------------------------------------------
 COPY ./requirements.txt /
+RUN yum install -y epel-release && \
+    yum install -y python34-devel python34-pip gcc make && \
```
Comment (abs51295) [inline Dockerfile]: We don't run code on python3 yet.

**Comment by abs51295** [comment]
@tisnik I am sending a PR to https://github.com/fabric8-analytics/fabric8-analytics-stack-analysis-base/blob/master/Dockerfile to add `make`.

**Comment by abs51295** [comment]
@tisnik https://github.com/fabric8-analytics/fabric8-analytics-stack-analysis-base/pull/6

**Comment by tisnik** [comment]
[test]

**Comment by tisnik** [comment]
[test]

**Comment by abs51295** [comment]
Merging this now.

**Pull request merged**